RELEASING.md

```diff
@@ -0,0 +1,62 @@
+# Releasing Kubeclient
+
+## Versioning
+Kubeclient release versioning follows [semvar](https://semver.org/).
+At some point in time it is decided to release version x.y.z.
+Prerequisite: A [changelog](CHANGELOG.md) is generated & commited to the release branch.
+
+```bash
+# USERNAME=bob
```

Comment:
why commented out? (you use it below)
maybe call it RUBYGEMS_USERNAME for clarity.